cmd/utils/flags.go

```diff
@@ -403,10 +403,6 @@ var (
 		Usage: "Number of CPU threads to use for mining",
 		Value: 0,
 	}
-	MinerNotifyFlag = cli.StringFlag{
```

Comment:
Maybe mention these flags' removal on the PR as "Other Changes". and/or maybe on the parent PR after this one is merged.

Reply:
done